doc/classes/ARVRServer.xml

```diff
@@ -8,8 +8,26 @@
 	</description>
 	<tutorials>
 		<link>https://docs.godotengine.org/en/3.3/tutorials/vr/index.html</link>
-	</tutorials>
+	</tutorials>add
 	<methods>
+		<method name="add_interface">
+			<return type="void">
+			</return>
+			<argument index="0" name="interface" type="ARVRInterface">
+			</argument>
+			<description>
+				Register a [ARVRInterface] object.
```

Comment:
Should be in 3rd person (and I think "an" before "AR"?) so:
```suggestion
				Registers an [ARVRInterface] object.
```

Same applies to other descriptions below.